io/disk/Scsi_add_remove.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2017 IBM
+# Author: Narasimhan V <sim@linux.vnet.ibm.com>
+# Author: Venkat Rao B <vrbagal1@linux.vnet.ibm.com>
+
+import time
+from avocado import Test
+from avocado.utils import process, genio
+from avocado.utils.software_manager import SoftwareManager
+
+
+class Scsi_add_remove(Test):
+
+    '''
+    Class to execute scsi add/remove operation
+    '''
+    def setUp(self):
+
+        '''
+        Function for preliminary set-up to execute the test
+        '''
+        sm = SoftwareManager()
+        if not sm.check_installed("lsscsi") and not sm.install("lspci"):
+            self.error("lsscsi is not installed")
+        self.pci_device = self.params.get("pci_device", default=None)
```

Comment:
since we are having None as default, we should skip here if pci_device is None. Else, we would hit TypeError in line 48.